nginx-controller/controller/controller.go

```diff
@@ -1222,6 +1229,19 @@ func (lbc *LoadBalancerController) getMinionsForMaster(master *nginx.IngressEx)
 			glog.Errorf("Ingress Resource %v/%v with the 'nginx.org/mergible-ingress-type' annotation set to 'minion' must contain a Path", ings.Items[i].Namespace, ings.Items[i].Name)
 			continue
 		}
+
+		for _, path := range ings.Items[i].Spec.Rules[0].HTTP.Paths {
+			for _, minion := range minions {
+				for _, minionPath := range minion.Ingress.Spec.Rules[0].HTTP.Paths {
+					if path.Path == minionPath.Path {
+						glog.Errorf("Ingress Resource %v/%v with the 'nginx.org/mergible-ingress-type' annotation set to 'minion' cannot contain the same path as another ingress resource, %v/%v",
+							ings.Items[i].Namespace, ings.Items[i].Name, minion.Ingress.Namespace, minion.Ingress.Name)
+						continue IngressLoop
+					}
```

Comment:
In this case, if an Ingress resource has a conflicting path and is not the oldest Ingress resource, the Ingress resource is completely Ignored. Maybe it is better to not ignore it, but only ignore that conflicting path?

Reply:
I agree, we can just log an error and not include the newest path. The configuration will also contain which Ingress Resource generates which path. This can also aid the user in debugging. I can also add more instructions on this feature.